app/Http/Controllers/PollController.php

```diff
@@ -58,15 +87,18 @@ public function show($poll)
      */
     public function update(Request $request, Poll $poll)
     {
-        $poll->update($request->only(['name','url']));
-
-        update($poll->users(), $request->usersToAdd);
-        update($poll->items(), $request->pollsToAdd);
-
-        update($poll->users(), $request->usersToRemove, true);
-        update($poll->items(), $request->pollsToRemove, true);
-
-        return new PollResource(Poll::with('users', 'items')->find($poll->id));
+        $is_admin = $poll->users->find(Auth::id())->pivot->admin == 1;
+        $ratings = DB::select('select * from poll_ratings where poll_id = ? AND user_id = ?', [$poll->id, Auth::id()]);
+        $is_not_empty = count($ratings) > 0;
+        error_log("admin : " . $poll->users->find(Auth::id())->pivot->admin);
+        if ($is_admin && $is_not_empty) {
+            $poll->update(['chosen_item_id' => 1]);
```

Comment:
chosen_item_id should be chosen randomly using the ratings of all items and not be hardcoded :eyes: